functions/ocr/app/ocr_test.go

```diff
@@ -0,0 +1,187 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ocr
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"strings"
+	"testing"
+
+	"cloud.google.com/go/pubsub"
+	"cloud.google.com/go/storage"
+	"cloud.google.com/go/translate"
+	vision "cloud.google.com/go/vision/apiv1"
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
```

Comment:
Sorry, we should get rid of this dependency and `os.Getenv("GOLANG_SAMPLES_PROJECT_ID")` manually. Can define a function to do the check to make it easy to call. Having this dependency means `github.com/GoogleCloudPlatform/golang-samples` shows up in the `go.mod` and gets downloaded whenever the function is deployed, which will slow deployments down.

Reply:
Done